tests/acceptance/pageObjects/personalPage.js

```diff
@@ -243,7 +243,7 @@ module.exports = {
         .click('@dialogConfirmBtnSecondaryEnabled')
         .waitForElementNotPresent('@dialog')
         .waitForAjaxCallsToStartAndFinish()
-        .waitForElementVisible('@fileUploadStatus')
+        .waitForElementVisible('@uploadSuccess')
```

Comment:
why check for success only?
```suggestion
        .waitForElementVisible('@uploadSuccess')
```

Reply:
i think this function is for confirm overwrite `confirmFileOverwrite`

Reply:
how does that fix the issue? any findings?

Reply:
Lets analysis this confirmFileOverwrite method .
```
      confirmFileOverwrite: async function () {
      await this.waitForAnimationToFinish() // wait for transition on the modal to finish==> (animation after selecting uploading resource)
        .waitForElementVisible('@dialogConfirmBtnSecondaryEnabled')
        .click('@dialogConfirmBtnSecondaryEnabled') ==>(click Replace button in dialogue)
        .waitForElementNotPresent('@dialog') ==>(checking dialogue box vanish after clicking Replace button)
        .waitForAjaxCallsToStartAndFinish() 
        .waitForElementVisible('@uploadSuccess') ===> after that checking upload  message dialogue b
        .closeFileFolderUploadProgress() ===> it close upload message dialogue box
      return this
    },
```
if we let the test move forward only after upload success then it has high chance that version will be created

Reply:
> if we let the test move forward only after upload success then it has high chance that version will be created

Yes indeed. we are closing the progress dialog on whether the upload fails or succeeds after all and with that it would be hard to tell where the test failed.
Just to be sure that the nightly failure is due to the file overwrite.